Documentation/gettingstarted/kubeproxy-free.rst

```diff
@@ -53,6 +53,15 @@ and the token returned by ``kubeadm init``:
 
    kubeadm join <..>
 
+Note: newer kernels do not include `br_netfilter` module, so you might need to disable netfilter preflight check
+when running your kubeadm `init` and `join` commands:
+
+.. code:: bash
+
+   kubeadm <..> --ignore-preflight-errors=FileContent--proc-sys-net-bridge-bridge-nf-call-iptables
+
+Cilium does not need `br_netfilter` module to operate in kube-proxy-less mode.
+
```

Comment:
The `Note:` should be done as `.. note::`. Would also formulate this more generically since Cilium in general does not depend on any bridge device or br_netfilter module. (So nothing todo with kube-proxy-less mode.)

Reply:
Be advised also, sphinx uses 3-space indentation for proper formatting in the page, so all new lines added below the `.. note::` would need to have that indentation.

Reply:
Thanks for feedback, TIL. Let me know how this looks now.